cppcache/include/geode/CacheableString.hpp

```diff
@@ -54,11 +54,11 @@ class APACHE_GEODE_EXPORT CacheableString
       : m_str(std::move(value)), m_hashcode(0) {
     bool ascii = isAscii(m_str);
 
-    m_type = m_str.length() > (std::numeric_limits<uint16_t>::max)()
-                 ? ascii ? DSCode::CacheableASCIIStringHuge
-                         : DSCode::CacheableStringHuge
-             : ascii ? DSCode::CacheableASCIIString
-                     : DSCode::CacheableString;
+    m_type =
+        m_str.length() > (std::numeric_limits<uint16_t>::max)()
+            ? ascii ? DSCode::CacheableASCIIStringHuge
+                    : DSCode::CacheableStringHuge
+            : ascii ? DSCode::CacheableASCIIString : DSCode::CacheableString;
```

Comment:
What is going on with formatting changes. There are several in this PR that don't appear related to the change at hand and some that don't seem to conform to the style guide.

Reply:
Fixed - just another artifact of the fact that the CI version of clangformat is "special" :/

Reply:
Did you make these changes on Windows? The CI version and macOS version seem to be in agreement.